.github/workflows/publish-firefox.yml

```diff
@@ -19,6 +20,7 @@ jobs:
         continue-on-error: true
         with:
           zipFilePath: yomitan-firefox.zip
+          sourcesZipFilePath: yomitan-${{ github.ref_name }}.zip
```

Comment:
At least on https://github.com/yomidevs/yomitan/releases/tag/24.11.11.0 the source code seems to be named `24.11.11.0.zip`, not `yomitan-24.11.11.0.zip`, do you think release-downloader prepends yomitan for some reason?